bin/cdxgen.js

```diff
@@ -73,6 +73,10 @@ const args = yargs(hideBin(process.argv))
     description:
       "Dependency track project id. Either provide the id or the project name and version together"
   })
+  .option("parentProjectId", {
```

Comment:
This must be specified using hyphen notation here.
```
.option("parent-project-id", {
```

Both hyphen and camel-cased versions would then become available args. No change is required to index.js

Reply:
And finally you could add this new argument to the readme file. The way to do it is to run

```
node bin/cdxgen.js -h
```

then copy paste the whole block correctly.